staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go

```diff
@@ -176,6 +177,9 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (*authenticator.R
 	*/
 
 	remaining := req.TLS.PeerCertificates[0].NotAfter.Sub(time.Now())
+	if remaining.Seconds() < 86400 {
```

Comment:
Short-lived certificates would *always* trip this logging, I don't think assuming a certificate shorter than a day is always bad. See https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2784-csr-duration#proposal, certificates as short as 10 minutes are permitted.